src/fetchLikes.js

```diff
@@ -0,0 +1,11 @@
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/ewWKj7zRfM05pYQRcA4r/likes/';
+const fetchLikes = (pokemon, item) => fetch(url)
+  .then((response) => response.json())
+  .then((likes) => {
+    likes.forEach((pkLike) => {
+      if (pkLike.item_id === pokemon.name) {
+        item.innerHTML = `${pkLike.likes} likes`;
+      }
+    });
+  });
```

Comment:
It's recommended to use `async/await` format to fetch and send data to the API. This will help you to avoid the callback hell which can lead to hard to debug errors. Please make the change.

Reply:
Hello 
Thanks for your feedback, if you check the displayPockes.js file and see the functionality, when we call the fetchlikes functions, that you comment on we use the async/await, so it gives us the same result if we use it in fitchlikes.js

Reply:
It is recommended to use async/await whenever it is appropriate when fetching API data. But in this particular case, the student has used a lighter code. I will make this comment optional. :pray: